app/src/sharedTest/java/org/oppia/app/home/HomeActivityTest.kt

```diff
@@ -132,8 +130,6 @@ class HomeActivityTest {
 
   @Test
   fun testHomeActivity_secondOpen_hasWelcomeBackString() {
```

Comment:
You can remove this entire test case and also `testHomeActivity_firstOpen_hasWelcomeString ` this test case

Reply:
Done :)